### ISSUE TITLE
No updates to the learning rate

### ISSUE THREAD

**Issue by samre12** [devsisters/DQN-tensorflow#53]
[This](https://github.com/devsisters/DQN-tensorflow/blob/c7b1f1051dfa152530322445fc8febb9a2ea078b/dqn/agent.py#L297) suggests that the maximum of the minimum learning rate and exponentially decayed rate is calculated. But in the [configurations file](https://github.com/devsisters/DQN-tensorflow/blob/c7b1f1051dfa152530322445fc8febb9a2ea078b/config.py#L14), both the learning rate and the minimum learning rate are supplied the same values. This will result in no updates to the learning rate with more training steps. 
**OR** Is this specifically for the case with no updates in the learning rate?
Thanks.
